packages/url_launcher/lib/url_launcher.dart

```diff
@@ -15,10 +15,38 @@ const MethodChannel _channel =
 /// The returned future completes with a [PlatformException] on invalid URLs and
 /// schemes which cannot be handled, that is when [canLaunch] would complete
 /// with false.
-Future<Null> launch(String urlString) {
+///
+/// [forceSafariVC] is only used in iOS. If unset, the launcher opens web URLs
+/// in the safari VC, anything else is opened using the default handler on the
+/// platform. If set to true, it opens the URL in the Safari view controller.
```

Comment:
Describe failure mode if not openable in safari?

Reply:
Done.